template-html5-app/src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>bs-example-html5</title>
```

Comment:
With the display of this html, it's more like a example rather than a template as specified in the folder route.